ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Util/LeakyBucket.hs

```diff
@@ -37,14 +41,16 @@ module Ouroboros.Consensus.Util.LeakyBucket (
 import           Data.Ratio ((%))
 import           Data.Time (DiffTime)
 import           Data.Time.Clock (diffTimeToPicoseconds)
+import           GHC.Generics (Generic)
+import           NoThunks.Class (NoThunks)
 import           Ouroboros.Consensus.Util.IOLike
-                     (MonadAsync (Async, async, uninterruptibleCancel),
+                     (MonadAsync (async, uninterruptibleCancel),
                      MonadCatch (handle), MonadDelay (threadDelay),
                      MonadFork (throwTo), MonadMask, MonadMonotonicTime,
                      MonadSTM, MonadThread (ThreadId, myThreadId),
                      MonadThrow (finally), SomeException, StrictTVar, Time,
-                     atomically, diffTime, getMonotonicTime, readTVar,
-                     readTVarIO, uncheckedNewTVarM, writeTVar)
+                     atomically, diffTime, getMonotonicTime, newTVarIO,
+                     readTVar, readTVarIO, writeTVar)
```

Comment:
We usually import this without an explicit import list, feel free to also do that if you are annoyed by the churn.

Reply:
7664b8b7bbd2557e1b563853e77a83278ce7306a